resource-detection-gcp/build.gradle.kts

```diff
@@ -30,7 +30,7 @@ dependencies {
     implementation("io.opentelemetry:opentelemetry-api")
     implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
     implementation("io.opentelemetry:opentelemetry-exporter-logging-otlp")
-    implementation("io.opentelemetry.contrib:opentelemetry-gcp-resources:1.34.0-alpha")
+    implementation("io.opentelemetry.contrib:opentelemetry-gcp-resources:1.42.0-alpha")
```

Comment:
I'll fix renovate so it will start auto-updating these

Reply:
#566

Reply:
Thank you @trask !